### PR TITLE
HRINT-365 Java Demo - Create Document

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

**Comment by topalach** [comment]
👍

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        newCompany.setPhone(companyPhone);
+
+        Contact newContact = new Contact();
+        newContact.setName(contactName);
+        newContact.setTitle(contactTitle);
+        newCompany.setContact(newContact);
+        //endregion
+
+        try (IDocumentSession session = DocumentStoreHolder.store.openSession()) {
+
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java]: remove this line

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        //endregion
+    }
+
+    public static class RunParams {
+        private String companyName;
+        private String companyPhone;
+        private String contactName;
+        private String contactTitle;
+
+        public String getCompanyName() { return companyName; }
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java]: did you auto generate this using Rider?

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/metadata.json [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+            "Slug" : "step-3",
+            "DescriptionLines": [
+                "* The Session takes care of generating the document identifier automatically.<br>",
+                "  The new stored entity ID is now available and can be used.",
+                "",
+                "* _If your Java class was defined with an 'id' property_:<br>",
+                "  => You can get the new stored entity ID as shown in the above example.",
+                "",
+                "* _If your Java class was Not defined with an 'id' property_:<br>",
+                "  => You can get the new stored entity using: `session.advanced.getDocumentId(newCompany)`"
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/metadata.json]: advanced()

Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/metadata.json]: done

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java (hunk 1 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java]: done

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java (hunk 2 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java]: Actually - no - I just typed :)
I know InteliJ has the 'Create getter and setter'
It seems that Rider doesn't have this option

Comment (ml054) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java]: oh I meant IntellijJ.

Please remove this code and regenerate to having standard formatting. 

In Rider there is no need to generate. C# syntax is more concise.

Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/createDocument/CreateDocument.java]: ok - pushed fix

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/common/models/Company.java [hunk 4]
```diff
@@ -10,58 +10,49 @@
     private String phone;
     private String fax;
 
-    public String getId() {
-        return id;
-    }
-
+    public String getId() {  return id;  }
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/common/models/Company.java]: ?

Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/common/models/Company.java]: fixed in latest commit

**Pull request merged**